From 74674c49a64abbdcdd5aeea016997af2959c6f05 Mon Sep 17 00:00:00 2001
From: Hodorgasm <nsane457@gmail.com>
Date: Mon, 3 Oct 2016 20:32:20 -0400
Subject: [PATCH] Add '#Include <vector>' to fix building on GCC-6

Don't rely on transitive header inclusion to declare std::vector as building with GCC-6 fails due to no '#Include <vector>' statement.

Upstream-Status: Backport [https://github.com/ros/ros_comm/commit/74674c49a64abbdcdd5aeea016997af2959c6f05]

This patch has been generated with:
  `git format-patch -1 74674c49a64abbdcdd5aeea016997af2959c6f05`
in the kinetic branch of the ros_comm repository.
This patch applies without modifications to the indigo release 1.11.20.

Signed-off-by: Lukas Bulwahn <lukas.bulwahn@oss.bmw-carit.de>
---
 tools/rosconsole/include/ros/console.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/rosconsole/include/ros/console.h b/tools/rosconsole/include/ros/console.h
index 4d2a86b..28901d7 100644
--- a/tools/rosconsole/include/ros/console.h
+++ b/tools/rosconsole/include/ros/console.h
@@ -40,6 +40,7 @@
 #include <cstdarg>
 #include <ros/macros.h>
 #include <map>
+#include <vector>
 
 #ifdef ROSCONSOLE_BACKEND_LOG4CXX
 #include "log4cxx/level.h"
-- 
2.5.5

